src/applications/pre-need/tests/00-all-fields.cypress.spec.js

```diff
@@ -0,0 +1,363 @@
+import Timeouts from 'platform/testing/e2e/timeouts';
+import testData from './schema/maximal-test.json';
+import cemeteries from './fixtures/mocks/cemeteries.json';
+
+describe('Pre-need test', () => {
+  // Test skipped to match Nightwatch
+  it.skip('fills the form and navigates accordingly', () => {
+    cy.intercept('POST', '/v0/preneeds/burial_forms', {
+      data: {
+        attributes: {
+          confirmationNumber: '123fake-submission-id-567',
+          submittedAt: '2016-05-16',
+        },
+      },
+    });
+    cy.intercept('POST', '/v0/preneeds/preneed_attachments', {
+      data: {
+        attributes: {
+          attachmentId: '1',
+          name: 'VA40-10007.pdf',
+          confirmationCode: 'e2128ec4-b2fc-429c-bad2-e4b564a80d20',
+        },
+      },
+    });
+    cy.intercept('GET', '/v0/preneeds/cemeteries', cemeteries);
+
+    cy.visit(
+      '/burials-and-memorials/pre-need/form-10007-apply-for-eligibility',
+    );
+    cy.get('body').should('be.visible');
+    cy.title().should(
+      'contain',
+      'Apply online for pre-need determination of eligibility in a VA National Cemetery | Veterans Affairs',
+    );
+    cy.get('.schemaform-title', { timeout: Timeouts.slow }).should(
+      'be.visible',
+    );
+    cy.get('.schemaform-start-button')
+      .first()
+      .click();
+
+    cy.url().should('not.contain', '/introduction');
+
+    cy.get('input[name="root_application_claimant_name_first"]', {
+      timeout: Timeouts.normal,
```

Comment:
You're removing all `Timeouts.normal`, right?

Reply:
Looks like the formatting outdid my site-wide search for these.  I'll do another pass :)